myuw/data/resource_link_import.csv

```diff
@@ -56,40 +56,40 @@ Services for Faculty and Staff,Research,all,https://www.washington.edu/research/
-Campus Life,Housing & Dining,all,,,https://www.hfs.washington.edu/dining/,Campus Dining,http://www.uwb.edu/food,Campus Dining,http://www.tacoma.uw.edu/food-services/locations,Campus Dining,no
+Campus Life,Housing & Dining,all,,,https://www.hfs.washington.edu/dining/,Campus Dining,http://www.uwb.edu/food,Campus Dining,https://www.tacoma.uw.edu/uwt/fa/facilities/food-services,Campus Dining,no
 Campus Life,Housing & Dining,all,https://www.hfs.washington.edu/myhfs/ledger.aspx,Check HFS Balance,,,,,,,no
 Campus Life,Housing & Dining,all,,,https://www.hfs.washington.edu/housing/,Student Housing,http://www.uwb.edu/housing,Student Housing,http://www.tacoma.uw.edu/housing,Housing,no
 Campus Life,Housing & Dining,all,,,https://facilities.uw.edu/,UW Facilities,https://www.uwb.edu/facility,Facilities and Campus Operations,http://www.tacoma.uw.edu/facilities-services/facilities-services,Facilities Services,no
-Campus Life,Housing & Dining,all,,,https://facilities.uw.edu/services/requests,Submit Work Request,http://dlweb.megamation.com/uwb/DLWEB.php/O4W_INFO_PAGE,Facilities Maintenance Request,http://www.tacoma.uw.edu/facilities-services/work-order-request,Facilities Services: Work Order Request,no
+Campus Life,Housing & Dining,all,,,https://facilities.uw.edu/services/requests,Submit Work Request,http://dlweb.megamation.com/uwb/DLWEB.php/O4W_INFO_PAGE,Facilities Maintenance Request,https://www.tacoma.uw.edu/uwt/fa/facilities/work-order-request,Facilities Services: Work Order Request,no
 Campus Life,Rules and Regulations,all,http://www.washington.edu/students/reg/ferpa.html,Family Educational Rights and Privacy Act (FERPA) – Privacy of Student Information,,,,,,,no
-Campus Life,Rules and Regulations,all,,,https://ap.washington.edu/ormp/,Office of Research Misconduct Proceedings,https://www.uwb.edu/academic-affairs/about/policies/studentconduct/student-guide,Student Guide to Academic Integrity,https://ap.washington.edu/ormp/,Office of Research Misconduct Proceedings,no
-Campus Life,Rules and Regulations,all,,,http://www.washington.edu/cssc/student-conduct-overview/student-code-of-conduct/,Student Conduct Code,http://www.uwb.edu/studentservices/studentconduct,Student Conduct and Responsibility,https://www.tacoma.uw.edu/catalog-two/student-rights-responsibilities,Student Rights and Responsibilities,no
+Campus Life,Rules and Regulations,all,,,https://ap.washington.edu/ormp/,Office of Research Misconduct Proceedings,https://www.uwb.edu/studentaffairs/studentconduct/student-guide,Student Guide to Academic Integrity,https://ap.washington.edu/ormp/,Office of Research Misconduct Proceedings,no
+Campus Life,Rules and Regulations,all,,,http://www.washington.edu/cssc/student-conduct-overview/student-code-of-conduct/,Student Conduct Code,http://www.uwb.edu/studentservices/studentconduct,Student Conduct and Responsibility,https://www.tacoma.uw.edu/uwt/drs/rights,Student Rights and Responsibilities,no
```

Comment:
I think the correct URL here should be https://www.tacoma.uw.edu/uwt/student-conduct/student-code
...the one you have is DRS-specific.

Reply:
And the label should be "Studnet Conduct Code"